collaboration/dispatch/__init__.py

```diff
@@ -6,3 +6,5 @@
 
 Heavily modified for Django's purposes.
 """
+
+from sugar3.dispatch.dispatcher import Signal
```

Comment:
`telepathyclient.py` uses `dispatch.Signal`.  You added an import from sugar3 module.  I'm not sure why.  It doesn't make sense to me yet.  This will prevent the file `collaboration/dispatch/dispatcher.py` from being used.  If that is your intention, then that file is not needed?  And nor are the other files in `collaboration/dispatch` and you could have changed the import in `telepathyclient.py`?  Can Turtle Art the application (not the Sugar activity) work without Sugar Toolkit installed?  I guess not, as there are a lot of imports.